en/developer_guide/block_override.rst

```diff
@@ -0,0 +1,123 @@
+Block Override
+==============
+
+Context
+-------
+
+From time to time, the block provided by Open Orchestra will fulfill some of your requirements
+but you would want to modify a little part of their behaviour.
+
+As the block is partially functional, you do not want to create an other block with the functionality.
+
+We provide you different ways to override part of the Open Orchestra block.
```

Comment:
It could be usefull to describe here the different use cases and the adapted solutions, so a developer can trigger easily and quickly its use case.